app/cli/cmd/attestation_add.go

```diff
@@ -47,6 +48,23 @@ func newAttestationAddCmd() *cobra.Command {
 		Annotations: map[string]string{
 			useWorkflowRobotAccount: "true",
 		},
+		Example: `  chainloop attestation add --attestation-id <attestation-id> --name <material-name> --value <material-value> --kind <material-kind> --annotation key=value,key2=value2
```

Comment:
I'd not add `--attestation-id <attestation-id>` since this is a feature not enabled by default, you will need to `init` with `remote-state`

Reply:
In fact, that doesn't seem an example, since kind and name are incompatible. If we want to show all the options, it might make more sense to use it somewhere else, is there an `usage` option?

Reply:
There is an `Example` fields where this is loaded. I'll remove the first option all together since you're right it can be confusing.